ocs_ci/deployment/deployment.py

```diff
@@ -225,6 +228,68 @@ def create_catalog_source(self):
         # Wait for catalog source is ready
         catalog_source.wait_for_state("READY")
 
+    def create_stage_operator_source(self):
+        """
+        This prepare operator source for OCS deployment from stage.
+        """
+        logger.info("Adding Stage Secret")
+        # generate quay token
+        credentials = {
+            "user": {
+                "username": config.DEPLOYMENT.get("stage_quay_username"),
```

Comment:
I wouldn't use .get method as this is mandatory parameter and this case we will silently continue with None value in username.

I would rather fail with key error here and user will directly see that those parameters has to be set.

Same below

Reply:
Fixed by 6ab36d8.